src/solver_info.jl

```diff
@@ -1,54 +1,21 @@
-using Pkg
 import MathProgBase
 export can_solve_mip, can_solve_socp, can_solve_sdp, can_solve_exp
-export set_default_solver, get_default_solver, isinstalled
+export set_default_solver, get_default_solver
 
 function set_default_solver(solver::MathProgBase.AbstractMathProgSolver)
   global DEFAULT_SOLVER
-  DEFAULT_SOLVER = solver
+  DEFAULT_SOLVER[] = solver
 end
 
 function get_default_solver()
-  if DEFAULT_SOLVER == nothing
+  if DEFAULT_SOLVER[] === nothing
     error("The default solver is set to `nothing`
-         You must have at least one solver installed to use Convex.
+         You must have at least one solver installed and loaded to use Convex.
          You can install a solver such as SCS by running:
```

Comment:
```suggestion
         You can load a solver such as SCS (once installed) by running 
         using SCS,
        before or after loading Convex.
```

It is probably not required to tell them how to install a package,
since if they try to load a package that is not installed, julia itself will tell them how to do a `Pkg.add`.
But it is perhaps worth explaining that loading is done with the `using` command (at least usually).